metadata_service/proxy/atlas_proxy.py

```diff
@@ -377,6 +402,8 @@ def get_table(self, *, table_uri: str) -> Table:
 
             columns = self._serialize_columns(entity=entity)
 
+            reports_guids = [report.get("guid") for report in attrs.get("reports") or list()]
```

Comment:
@dechoma is it possible to filter out the active reports here already ? I think you'll have the active status available here, so no need to make a query with the inactive queries.

Reply:
Unfortunately, attributes contain only report typeName and guid`{'owner': 'testuser', 'temporary': False, 'reports': [{'guid': '50160c85-cdd1-4bac-800a-9bb623b816d1', 'typeName': 'Report'}], ...() }`